src/views/NoteView.vue

```diff
@@ -23,50 +21,69 @@
 </template>
 
 <script setup lang="ts">
-/*
-import{ref, onMounted} from 'vue';
-import { useRoute } from 'vue-router';
-
-interface Note {
-  revision: string;,
-  channel: string;,
-  permission: string;,
-  body: string;
-}
+import { NoteSummary } from '@/types/api';
+import { ref, computed } from 'vue';
+
+const renderedMarkdown = computed(() => {
+  if (!note.value.summary) {
+    return '';
+  }
+  // markedでMarkdownをHTMLに変換し、DOMPurifyでサニタイズする
+  const rawHtml = marked(note.value.summary);
+  return DOMPurify.sanitize(rawHtml);
+});
```

Comment:
僕のコードに不備がありました、ごめんなさい！

```suggestion
const renderedMarkdown = computed(() => {
  if (!note.value.summary) {
    return '';
  }
  // markedでMarkdownをHTMLに変換し、DOMPurifyでサニタイズする
  const rawHtml = marked(note.value.summary) as string;
  return DOMPurify.sanitize(rawHtml);
});
```